.github/workflows/build_packages.yaml

```diff
@@ -488,7 +526,7 @@ jobs:
       with:
         images: ${{ github.repository_owner }}/${{ matrix.profile }}
         flavor: |
-          latest=${{ !github.event.release.prerelease }}
+          latest=${{ github.event_name == 'release' && !github.event.release.prerelease }}
           suffix=-elixir
```

Comment:
```suggestion
          suffix=-${{ matrix.arch }}-elixir
```
?

Reply:
`${{ matrix.arch }}` should be the last one. so it will be `suffix=-elixir-${{ matrix.arch }}`

Sorry I missed it while rebase, but what is the point to have a specific docker meta job for elixir? Can we have one common job and just use a matrix?

Reply:
Well, there were a couple difficulties I had that made me create those disjoint steps for Elixir:

- After discussing with @zmstone about the docker tags, I understood that we shouldn't use `emqx/emqx-elixir` for the image that contains Elixir, as it would mean a different repo than `emqx`.  So, it had to be in the tag instead of the image name.  That complicated a bit the direct use of the matrix.
- One label (`org.opencontainers.image.elixir.version`) should only be added for the Elixir tag.

Perhaps there's a smarter way to do it without splitting the steps, but I didn't find it initially.  :disappointed:

Reply:
thanks for clarification. Now I used some hacks to make a common job for both rebar and elixir build.

Now the whole workflow is huge and very complicated with a huge context that is not easy to read....